addons/collapse-footer/addon.json

```diff
@@ -56,6 +56,12 @@
         "addonEnabled": ["infinite-scroll"]
       },
       "default": false
+    },
+    {
+      "name": "Toggle on click",
```

Comment:
Would it make sense to turn this into a select?

Reply:
It only has 2 options, it's not subject to get more, and it's consistent with the other settings.